dts/bindings/ipc/zephyr,ipc-openamp-static-vrings.yaml

```diff
@@ -0,0 +1,30 @@
+# Copyright (c) 2021 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: OpenAMP (RPMsg with static VRINGs) backend
+
+compatible: "zephyr,ipc-openamp-static-vrings"
+
+include: base.yaml
+
+properties:
+  role:
+    description: OpenAMP roles
+    required: true
+    type: string
+    enum:
+      - host
+      - remote
+
+  shm:
```

Comment:
regarding https://github.com/zephyrproject-rtos/zephyr/blob/main/include/linker/devicetree_reserved.h#L121 and Linux DT implementation , `memory-region` seems to me more generic than `shm`

Reply:
good idea